packages/node_exporter/build.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+from subprocess import run
+import requests
+import tarfile
+
+response = requests.get("https://api.github.com/repos/prometheus/node_exporter/releases/latest")
```

Comment:
Can we please select a discrete version, so we always know which version ended up in out image?

Reply:
done